mon-pix/app/templates/components/password-reset-demand-form.hbs

```diff
@@ -1,54 +1,54 @@
 <div class="sign-form__container">
 
-  <a href={{homeUrl}} class="pix-logo__link" title="Lien vers la page d'accueil de Pix">
-    <img class="pix-logo__image" src="{{rootURL}}/images/pix-logo.svg" alt="Pix">
+  <a href={{this.homeUrl}} class="pix-logo__link" title="Lien vers la page d'accueil de Pix">
+    <img class="pix-logo__image" src="{{this.rootURL}}/images/pix-logo.svg" alt="Pix">
   </a>
 
   <div class="sign-form__header">
-    {{#unless _displaySuccessMessage}}
+    {{#unless this.displaySuccessMessage}}
       <h1 class="sign-form-title">Mot de passe oublié ?</h1>
       <div class="sign-form-header__instruction sign-form-header__instruction--short sign-form-subtitle">
         Entrez votre adresse e-mail ci-dessous, et c'est repartix
       </div>
     {{/unless}}
   </div>
 
-  {{#if _displayErrorMessage}}
+  {{#if this.displayErrorMessage}}
     <div class="sign-form__notification-message sign-form__notification-message--error" aria-live="polite">
       Cette adresse e-mail ne correspond à aucun compte
     </div>
   {{/if}}
 
-  {{#unless _displaySuccessMessage}}
-    <form {{action "savePasswordResetDemand" on="submit"}} class="sign-form__body">
+  {{#if this.displaySuccessMessage}}
+    <div class="sign-form-header__instruction sign-form-subtitle">Demande de réinitialisation de mot de passe</div>
+
+    <div class="password-reset-demand-form__body" aria-live="polite">
+      <span class="password-reset-demand-body__text sign-form-text">Un e-mail contenant la démarche à suivre afin de réinitialiser votre mot de passe
+        vous a été envoyé à l’adresse e-mail {{this.email}}.</span>
+      <span class="password-reset-demand-body__text sign-form-text">Si vous ne recevez pas cet e-mail, vérifiez vos courriers indésirables.</span>
+    </div>
+
+    <div class="password-reset-demand-form__home-link">
+      <a href={{this.homeUrl}} class="link" title="Lien vers la page d'accueil de Pix">Retour à l'accueil</a>
+    </div>
+  {{else}}
+    <form {{on 'submit' this.savePasswordResetDemand}} class="sign-form__body">
 
       <div class="sign-form-body__input">
-        <FormTextfield @label="Adresse e-mail" @textfieldName="email" @validationStatus="default" @inputBindingValue={{email}} />
+        <FormTextfield @label="Adresse e-mail" @textfieldName="email" @validationStatus="default"
+                       @inputBindingValue={{email}} />
```

Comment:
Pas de this.email ici. C'est normal ou c'est un oubli ?

Reply:
c'est un oubli